Unable to build bindings with msvc -- vcpkg.json missing from repo?

### System Info

Windows 11, VS 2022

### Information

- [X] The official example notebooks/scripts
- [ ] My own modified scripts

### Reproduction

Due to #1520, I am attempting to build the packages myself, but it is failing with dependency errors with msvc, they are not mentioned in the documentation, and there is no `vcpkg.json` in the repo to install them.

Notably, it appears that Vulkan_LIBRARY is missing:
![image](https://github.com/nomic-ai/gpt4all/assets/5217366/91fcde41-1a68-4b5f-bef2-1c408586e8c2)
```
CMake Error at C:/Program Files/Microsoft Visual Studio/2022/Enterprise/Common7/IDE/CommonExtensions/Microsoft/CMake/CMake/share/cmake-3.26/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find Vulkan (missing: Vulkan_LIBRARY Vulkan_INCLUDE_DIR glslc)
  (found version "")
Call Stack (most recent call first):
  C:/Program Files/Microsoft Visual Studio/2022/Enterprise/Common7/IDE/CommonExtensions/Microsoft/CMake/CMake/share/cmake-3.26/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
  C:/Program Files/Microsoft Visual Studio/2022/Enterprise/Common7/IDE/CommonExtensions/Microsoft/CMake/CMake/share/cmake-3.26/Modules/FindVulkan.cmake:597 (find_package_handle_standard_args)
  llama.cpp.cmake:158 (find_package)
  CMakeLists.txt:46 (include)
```

### Expected behavior

If there are additional dependencies needed to build the software, I would expect there to be a vcpkg.json file to install the vc packages somewhere in the repo.

--- a/gpt4all-bindings/csharp/README.md
+++ b/gpt4all-bindings/csharp/README.md
@@ -23,6 +23,12 @@ gpt4all-bindings/
           └── linux-x64
 ```
 
+## Prerequisites
+
+On Windows and Linux, building GPT4All requires the complete Vulkan SDK. You may download it from here: https://vulkan.lunarg.com/sdk/home
+
+macOS users do not need Vulkan, as GPT4All will use Metal instead.
+
 ## Local Build Instructions
 > **Note** 
 > Tested On:

--- a/gpt4all-bindings/python/README.md
+++ b/gpt4all-bindings/python/README.md
@@ -15,6 +15,14 @@ pip install gpt4all
 
 ## Local Build Instructions
 
+### Prerequisites
+
+On Windows and Linux, building GPT4All requires the complete Vulkan SDK. You may download it from here: https://vulkan.lunarg.com/sdk/home
+
+macOS users do not need Vulkan, as GPT4All will use Metal instead.
+
+### Building the python bindings
+
 **NOTE**: If you are doing this on a Windows machine, you must build the GPT4All backend using [MinGW64](https://www.mingw-w64.org/) compiler.
 
 1. Setup `llmodel`

--- a/gpt4all-bindings/python/docs/gpt4all_typescript.md
+++ b/gpt4all-bindings/python/docs/gpt4all_typescript.md
@@ -58,6 +58,8 @@ const fltArray = createEmbedding(model, "Pain is inevitable, suffering optional"
 *   (win) msvc version 143
     *   Can be obtained with visual studio 2022 build tools
 *   python 3
+*   On Windows and Linux, building GPT4All requires the complete Vulkan SDK. You may download it from here: https://vulkan.lunarg.com/sdk/home
+*   macOS users do not need Vulkan, as GPT4All will use Metal instead.
 
 ### Build (from source)
 

--- a/gpt4all-bindings/typescript/README.md
+++ b/gpt4all-bindings/typescript/README.md
@@ -58,6 +58,8 @@ const fltArray = createEmbedding(model, "Pain is inevitable, suffering optional"
 *   (win) msvc version 143
     *   Can be obtained with visual studio 2022 build tools
 *   python 3
+*   On Windows and Linux, building GPT4All requires the complete Vulkan SDK. You may download it from here: https://vulkan.lunarg.com/sdk/home
+*   macOS users do not need Vulkan, as GPT4All will use Metal instead.
 
 ### Build (from source)
 

--- a/gpt4all-chat/build_and_run.md
+++ b/gpt4all-chat/build_and_run.md
@@ -6,7 +6,7 @@ gpt4all-chat from source.
 
 ## Prerequisites
 
-On Windows and Linux, GPT4All requires Vulkan. You may download it from here: https://vulkan.lunarg.com/sdk/home
+On Windows and Linux, building GPT4All requires the complete Vulkan SDK. You may download it from here: https://vulkan.lunarg.com/sdk/home
 
 macOS users do not need Vulkan, as GPT4All will use Metal instead.
 

